packages/pieces/community/notion/src/lib/actions/get-page-or-block-children.ts

```diff
@@ -30,12 +51,22 @@ export const getPageOrBlockChildren = createAction({
       notionVersion: '2022-02-22',
     });
 
-    return getBlockChildrenRecursively(
-      notion,
-      context.propsValue.parentId,
-      context.propsValue.depth,
-      0
-    );
+    if (context.propsValue.markdown) {
+      const n2m = new NotionToMarkdown({
+        notionClient: notion,
+        config: { parseChildPages: false },
+      });
+      return n2m.toMarkdownString(
+        await n2m.pageToMarkdown(context.propsValue.parentId)
+      ).parent;
+    } else {
+      return getBlockChildrenRecursively(
+        notion,
+        context.propsValue.parentId,
+        context.propsValue.dynamic?.props.depth,
```

Comment:
You can access dynamic property field by `context.propsValue.dynamic['depth']`

Reply:
Thanks, changed it!